source/graphql/migrate-apollo.txt

```diff
@@ -0,0 +1,74 @@
+.. _migrate-apollo:
+
+Migrate GraphQL to Apollo
+=========================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to Apollo Server.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and 
+   `Apollo Server <https://www.apollographql.com/docs/apollo-server/>`_ for the
```

Comment:
Nit: this page contains many external links to third-party content that we don't control. IMO that is an invitation to get a bunch of broken links over time. Any chance we can remove some/most of these links and only link out to one or two of the most relevant places? i.e. instead of linking to all the individual https://www.apollographql.com/docs/ destinations, maybe we could link once to Apollo docs?

(Haven't looked yet but I'm assuming this same feedback may apply to the other guides.)

Reply:
The Apollo team wrote the guide and added these hyperlinks throughout, and they are pretty general docs links to pages that I don't see going away (e.g. Intro to Apollo Server, Intro to Apollo Client, etc) or are collaborative Apollo <> MDB blogs that they'd likely flag to us before removing / changing.

Apollo is aware of this migration guide and would want new customers, so I doubt they'd remove any of these links and at most, this guide would be up for only a year (until the EOL date). Would that make it ok to keep these links?

Reply:
Yeah, considering these guides will only be relevant for a year, I'm not too worried about URL drift. It's possible, but I don't think it's likely.